proprietary/design-tokens/src/component/utrecht/breadcrumb.tokens.json

```diff
@@ -27,19 +30,42 @@
         }
       },
       "link": {
+        "active": {
+          "text-decoration": {
+            "value": "underline"
+          }
+        },
         "background-color": {
           "value": "{utrecht.color.grey.90}"
         },
         "color": {
           "value": "{utrecht.color.black}"
         },
+        "hover": {
+          "color": {},
+          "text-decoration": {
+            "value": "underline"
+          },
+          "text-decoration-thickness": {
+            "value": "1px"
+          }
+        },
         "focus": {
           "background-color": {
             "value": "{utrecht.color.yellow.60}"
           },
           "color": {
             "value": "{utrecht.color.black}"
+          },
+          "text-decoration": {
+            "value": "underline"
+          },
+          "text-decoration-thickness": {
+            "value": "1px"
           }
+        },
+        "text-decoration": {
+          "value": "underline"
         }
```

Comment:
I'm not exactly sure why this change for the benefit of RVO would require token changes for Utrecht. If it is necessary to maintain existing behavior, that would be a good indication this is a breaking change.